stemcell_builder/stages/system_azure_wala/apply.sh

```diff
@@ -26,22 +27,18 @@ run_in_chroot $chroot "
   tar zxvf wala.tar.gz
   cd WALinuxAgent-${wala_release}
   sudo python setup.py install --skip-data-files
-  cp bin/waagent /usr/sbin/waagent
+  cp bin/* /usr/sbin/
```

Comment:
Why does this need to use `*`? If the only file that needs copying is `waagent2.0`, `cp bin/waagent*` would work.

Reply:
resolved.